main.go

```diff
@@ -124,7 +125,9 @@ func configureHealthChecks(ctx context.Context,
 		hasErrors = true
 	}
 
-	elasticClient := elastic.NewClientWithHTTPClient(cfg.ElasticSearchAPIURL, cfg.SignElasticsearchRequests, elasticHTTPClient)
+	var indexes string
```

Comment:
Do you need to change this? You are supplying an empty string as the index to check.

Reply:
In the case of this API the search indexes that are built are dynamic, on the ticket there isn't a definition of what index to look up as this will vary depending on the environment (either local or not). 

A bit of background: The indexes are named after the instance ID generated for a dataset and hence cannot be determined upfront.

I feel further discussion on the list to be used for this particular service maybe needed. Or simply the check on being able to talk to an index is of little to no value in this service.

Reply:
I guess for now we should remove `var indexes string` and update the function like so:

`elasticClient := elastic.NewClientWithHTTPClient(cfg.ElasticSearchAPIURL, cfg.SignElasticsearchRequests, elasticHTTPClient)`

Which you can do as the indexes is a variadic variable (optional)